compiler/luci/pass/src/FuseGeluPass.cpp

```diff
@@ -95,20 +95,10 @@ class GeluPattern1 final : public GeluPatternBase
     _mul_half = candidate;
   }
 
-public:
-  bool matched();
+  ~GeluPattern1() override = default;
```

Comment:
Q) is this necessary ?

Reply:
Actually no, just for readability. Removed it